packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -185,11 +185,16 @@ export class MetriportMedicalApi {
    *
    * @param data The data to be used to create a new patient.
    * @param facilityId The facility providing the NPI to support this operation.
+   * @param externalId Optional external ID to be associated with the patient.
    * @return The newly created patient.
    */
-  async createPatient(data: PatientCreate, facilityId: string): Promise<PatientDTO> {
+  async createPatient(
+    data: PatientCreate,
+    facilityId: string,
+    externalId?: string
```

Comment:
Why don't just add `externalId` as part of the patient? That's how we store it on our DB, this is how we we return as response of requests (https://github.com/metriport/metriport/pull/1170/files/060358a76ad187e9bdc70e4500c94945d35ac7fc..fee8f0583e70b320ee54024a295698cd8477fd36#diff-9b502a20ecefc7049457e8b146dccf1750f41eafcd26196cebee632df15d9b20R13).